capstone/client/src/components/styles.css

```diff
@@ -318,6 +327,8 @@ h6 {
   margin-bottom: 10px;
   margin-left: 5px;
   margin-right: 5px;
+  background-color : #05386b;
```

Comment:
You can also try using the light blue from the Welcome page (#77bce0) for the "Show more information" and "Get directions" buttons.

Reply:
I tried this and it looked a teensy bit funky, I will send you a screenshot and you tell me what you think